README.md

```diff
@@ -0,0 +1,20 @@
+# firefly_client
+
+Python API for Firefly, IPAC's Advanced Astronomy Web UI Framework
+
+## Usage
+
+The client must be connected to a Firefly server.
+
+```
+from firefly_client import FireflyClient
+fc = FireflyClient('localhost:8080', 'mychannel')
+```
+
+A FITS image may be uploaded and displayed.
+
+```
+fval = fc.upload_file('image.fits')
+fc.show_fits(fval, 'myimage')
+
+
```

Comment:
I agree, a reference here would be good. Maybe also good to point to the release area for those wanting the standalone server.